content/developer/writing_widgets/widget-apis.md

```diff
@@ -302,7 +302,7 @@ Redirects user to parent page (used when you are in drill-down page).
 
 Shows/hides a loading spinner in widget header.
 
-**Not allowed in `render()` and `postRender()`** methods as it changes store state leading to `render()` and `postRender()` re-run.
+**Not allowed in `render()`** method as it changes store state leading to `render()` re-run.
```

Comment:
In case the intent is to remove all information about `postRender`, then I found some more occurrences:

```sh
13:15 $ rg -i postrender content
content/developer/writing_widgets/widget-definition.md
48:`fetchUrl`             | string/object | No | -            | If `fetchUrl` exists, the data from the URL is fetched by the application and passed to the render and postRender methods. To fetch multiple URLs, you must pass an object where the key is a name you select for this data, and the value is the URL. It is important to note that the render is called once before the data is fetched (to enable information about loading or partial data can be displayed) and once after the data is fetched.
400:### postRender(container, widget, data, toolbox)
404:`postRender` is called immediately after the widget has been made visible in the {{< param cfy_console_name >}}.
```

Is `postRender` deprecated? I don't think we are removing it :thinking: It's still there in the types

![image](https://user-images.githubusercontent.com/889383/127483038-0317af69-3f00-47a1-9ae9-92e67839acc7.png)

Reply:
`postRender` should not be removed from the documentation. As @Gelio pointed, it is still in use.

@stepek , you probably was misled by the description I provided in the RD-2775 (as it was created when planned to remove it and I didn't track it afterwards).